components/card/card_header.tsx

```diff
@@ -7,13 +7,44 @@ import classNames from 'classnames';
 type Props = {
     children: React.ReactNode;
     expanded?: boolean;
+    title?: React.ReactNode;
+    subtitle?: React.ReactNode;
+    buttonText?: React.ReactNode;
+    onClick?: () => void;
 };
 
 const CardHeader: React.FC<Props> = (props: Props) => {
     return (
         <div className={classNames('Card__header', {expanded: props.expanded})}>
-            {props.children}
-            {props.expanded && <hr className='Card__hr'/>}
+            {
+                props.title && props.subtitle ?
```

Comment:
I dont know why but this feels odd to me. If this component is already accepting children, meaning we can pass in anything to fill it up with content, why do we provide this alternate.

What about if we make a functional component that accepts these params and can be passed into CardHeader? 
It can look something like
```
<Card.header>
       <CardHeaderContent
             title=fsa
             subtitle=fdsaf
             onClick={void}
       />
</Card.header>
```
I would also say make the title required, subtitle optional, and onClick optional too incase we want a bare minimum of just a title in the card header.

My thoughts though .... Let me know what you guys think.